src/constants/index.tsx

```diff
@@ -450,7 +451,7 @@ export const NETWORK_DETAIL: { [chainId: number]: NetworkDetails } = {
       symbol: Currency.ETHER.symbol || 'ETH',
       decimals: Currency.ETHER.decimals || 18,
     },
-    rpcUrls: ['https://mainnet.optimism.io'],
+    rpcUrls: ['https://opt-mainnet.g.alchemy.com/v2/6cRVjVO2uOTC9gWFCsBnquUwOM9zuWQZ'],
```

Comment:
I noticed this issue on the SDK too. Let's avoid hardcoding RPC URLs

Reply:
what do you suggest?

Reply:
I would move them to environment variables like in Nimi :)